bearer/bearer.go

```diff
@@ -351,10 +354,10 @@ func (b Token) SigningKeyBytes() []byte {
 	return nil
 }
 
-// ResolveIssuer resolves issuer's user.ID from the key used for Token signing.
-// Returns zero user.ID if Token is unsigned or key has incorrect format.
+// ResolveIssuer resolves issuer's [user.ID] from the key used for Token signing.
```

Comment:
`[Token]`?

Reply:
It is an interesting question. Because this method is connected to the Token type. We _already here_ on the documentation page. Do we need lint to the type itself?

Updated to `[]` link, but we should decide in general, how we should handle these situations. Both variants are okay for me, but just do the same in all places